### ISSUE TITLE
[UI] Character and Light Cone Level Editing Remnants Present

### ISSUE THREAD

**Issue by WolfBungalow** [fribbels/hsr-optimizer#286]
## Description
Based on Update 03-26-2024 changelog, it seems like character and light cone levels other than 80 are no longer supported. The "Edit character" window gives the impression that they still are. Should these edit options be removed? Changes made to character and light cone levels seem to save, as the updated levels will show on the characters showcase page, but changes do not affect base stats and are not reflected in the "Edit character" window when reopened.

## Steps to Reproduce

1. Open in the "Characters > Character actions > Edit character" window.
2. Make a change to the character and or light cone level.
3. Click the save button.
4. Character showcase levels have changed.
5. Reopen the "Characters > Character actions > Edit character" window.
6. "Edit character" window levels have reverted to 80.

## Expected Behavior
Assuming level other than 80 are no longer supported then should these edit options be removed?

## Actual Behavior
Options to change character and light cone levels exist in "Edit character" window but do almost nothing.

## Screenshots
![image](https://github.com/fribbels/hsr-optimizer/assets/25573424/c62e4e37-922d-415a-800e-f7cd69b55553)

## Additional Information
None.

**Comment by fribbels** [comment]
Yeah this is a fair change, I don't see any use in keeping these anymore. Thanks for the reminder!

**Comment by WolfBungalow** [comment]
Depending on your future plans the character edit window and option from the characters tab may not be needed. The settings are available in the character optimization page.

![image](https://github.com/fribbels/hsr-optimizer/assets/25573424/a0ceef91-08bd-4103-9e16-f5c346177438)

**Comment by fribbels** [comment]
Released in https://github.com/fribbels/hsr-optimizer/pull/298
